docs/spec/api.md

```diff
@@ -332,7 +332,7 @@ digest. The _hex_ portion is the hex-encoded result of the hash.
 We define a _digest_ string to match the following grammar:
 ```
 digest      := algorithm ":" hex
-algorithm   := /[A-Fa-f0-9_+.-]+/
+algorithm   := /[A-Za-z0-9_+.-]+/
```

Comment:
for reference; corresponding PR in the distribution-spec; https://github.com/opencontainers/distribution-spec/pull/61